android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -231,6 +232,21 @@ public void sendNotification(Bundle bundle) {
                 notification.setVibrate(new long[]{0, vibration});
             }
 
+            int badge = -1;
+            if (bundle.containsKey("badge")) {
```

Comment:
If we want iOS parity, `badge` is right key to read. The doc linked is pretty clear, and we can confirm. We use `badge` payload attribute (along side `alert`, `content-available`, and `sound` -- also discussed in the doc)  for our existing iOS-only app.